Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.10
+FROM docker.io/library/alpine:3.20
```

Comment:
might be able to remove `docker.io/library/`, I was having issues building in WSL2, but I think I fixed my root issue, if you could try without the prefix, that would be grand.